### ISSUE TITLE
Requires Modern Runtime

### ISSUE THREAD

**Issue by jablair** [mierau/DMTemplates#1]
As written, DMTemplates can't be used in an app that has to support non-64 bit Macs, because it uses synthesized ivars. It looks like declaring ivars for each property is enough to regain 32-bit support.

**Comment by mierau** [comment]
You're right. I'd be happy to make this fix this evening, or if you want to commit a fix, I can pull it in.

Thanks so much!

Best,
Dustin

**Comment by jablair** [comment]
Had to wait till I got home from work. Not a major change, clearly, if you'd already started it.
